ember-cli-build.js

```diff
@@ -49,18 +55,7 @@ module.exports = function(defaults) {
     hinting: process.env.EMBER_CLI_TEST_COMMAND || !isProductionLikeBuild
   });
 
-  // Use `app.import` to add additional libraries to the generated
-  // output files.
-  //
-  // If you need to use different assets in different
-  // environments, specify an object as the first parameter. That
-  // object's keys should be the environment name and the values
-  // should be the asset to use in that environment.
-  //
-  // If the library that you are including contains AMD or ES6
-  // modules that you would like to import into your application
-  // please specify an object with the list of modules as keys
-  // along with the exports of each module as its value.
+  app.import('node_modules/normalize.css/normalize.css');
```

Comment:
This is how css is imported into `vendor.css`. If we have a need for a sass import, we can use `normalize-sass`